reactive/kubernetes_worker.py

```diff
@@ -900,39 +909,61 @@ def get_kube_api_servers(kube_api):
 
 @when('nrpe-external-master.available')
 @when_not('nrpe-external-master.initial-config')
-def initial_nrpe_config(nagios=None):
+def initial_nrpe_config():
```

Comment:
what's the point of this handler? if we're gonna call `update_nrpe_config` anyway, why not just let the gates on that function do the right thing?  as is, we're allowing nrpe checks when kube-api and kube-control may not be available.  Seems like we'll get false positives until those flags are set.

Reply:
handled in #42